built_value_generator/lib/src/serializer_source_library.dart

```diff
@@ -133,11 +158,27 @@ abstract class SerializerSourceLibrary
 
   bool get needsBuiltJson => sourceClasses.isNotEmpty;
 
+  Iterable<String> computeErrors() {
+    var result = <String>[];
+
+    if (wrongSerializersDeclarations.isNotEmpty) {
+      result.add(
+          'These top level getters are annotated @SerializersFor but do not '
+          'have the required type Serializers, please fix the type or remove '
+          'the annotation: ${wrongSerializersDeclarations.join(', ')}');
+    }
+
+    return result;
+  }
+
   /// Generates serializer source for this library.
   String generateCode() {
-    var errors = concat(sourceClasses
-        .map((sourceClass) => sourceClass.computeErrors())
-        .toList());
+    var errors = [
+      ...computeErrors(),
+      ...concat(sourceClasses
+          .map((sourceClass) => sourceClass.computeErrors())
+          .toList())
```

Comment:
you can do expand instead of concat(map), or with fancy literals something like this:

```
for (var sourceClass in sourceClasses)
   ...sourceClass.computeErrors()
```

Reply:
Done, thanks!